portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin:0;
+}
+
+
+#fullpage{
+    min-width: 100vw;
```

Comment:
[alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order), here and elsewhere